blocks/products/includes/renderer.php

```diff
@@ -59,13 +65,14 @@ function products_block_render_callback( $block_attributes) {
  * Render Product
  * 
  * @param stdClass $product
+ * @param bool $display_byline
  * @return string
  */
-function render_product( stdClass $product ) : string {
+function render_product( stdClass $product , bool $display_byline) : string {
 	$link   = esc_url( apply_filters( 'cata_product_block_link', $product->permalink ) );
 	$title  = esc_html( $product->name );
 	$price  = render_price( $product );
-	$byline = render_byline( $product );
+	$byline = render_byline( $product, $display_byline );
```

Comment:
Could this be a ternary instead of passing the parameter to render byline? For some reason its weird to me to pass a function a parameter that tells it to do nothing.

Reply:
I see what you're saying, I just am not sure what you mean by using a ternary?

Reply:
`$byline = true === $display_byline ? render_byline( $product ) : '';`

Reply:
Ah gotcha, i was thinking of a ternary IN the string literal.

Updated!